NEWS.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
Personally I'd not bother with NEWS until we start iterating with a package that's generally useful but it's up to you